src/main/java/uk/gov/hmcts/reform/pip/account/management/controllers/AccountController.java

```diff
@@ -241,4 +246,57 @@ public ResponseEntity<Void> deleteExpiredIdamAccounts() {
         accountVerificationService.findIdamAccountsForDeletion();
         return ResponseEntity.noContent().build();
     }
+
+    @ApiResponses({
+        @ApiResponse(responseCode = OK_CODE, description = "Page data with a list of Pi User objects"),
+        @ApiResponse(responseCode = AUTH_ERROR_CODE, description = NOT_AUTHORIZED_MESSAGE)
+    })
+    @Operation(summary = "Get all accounts except third party in a page with filtering")
+    @GetMapping("/all")
+    public ResponseEntity<Page<PiUser>> getAllAccountsExceptThirdParty(
+        @RequestParam(name = "pageNumber") int pageNumber,
```

Comment:
Might be worth adding some defaults to number / size just for completeness

Reply:
Done :) has to be a string as thats all defaultValue supports but will be converted to int.